sdk/storage/azure-storage-blob-cryptography/pom.xml

```diff
@@ -59,12 +59,6 @@
       <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
@alzimmermsft & @JimSuplizio I'd like to get your opinion on this change. I wasn't sure why we ended up with this dependency being listed twice. So I removed one.

Reply:
This likely predates some other changes made to our POMs, thank you for removing it 😃

Reply:
@mitchdenny I'd actually had this change in the PR I'm working on as well which is delayed due to ongoing docs fun.